quickstart-node/src/middlewares/error.middleware.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express';
+import { LoggerService } from '@services/monday-code';
+
+/** Here you can put your error handling logic
+ * NOTE: we are using express 5, so each error, even async errors, will get here eventually,
+ *       so no need for next(error) in the middlewares before.
+ *       Feel free to go to this page and learn more about error handling in express: https://expressjs.com/en/guide/error-handling.html
+ */
+const errorHandler = (err: any, req: Request, res: Response, next: NextFunction) => {
+  const statusCode = err.status || 500;
+  const message = err.message || 'Internal Server Error';
+
+  if (statusCode >= 500) {
+    LoggerService.getInstance().error(message, err);
```

Comment:
Did you check about the infinite error loop we talked about?

Reply:
Yes, it does not crashes, and the server can keep on getting requests